clientv3/watch.go

```diff
@@ -634,8 +634,16 @@ func (w *watchGrpcStream) run() {
 		case ws := <-w.closingc:
 			w.closeSubstream(ws)
 			delete(closing, ws)
+			// check if there are any non-nil resuming watcher streams left
+			resumingNonNil := false
```

Comment:
Move this logic into a w.HasResumingWatchers() function?

Reply:
@jpbetz done